pom.xml

```diff
@@ -821,7 +821,8 @@
               <groupId>org.apache.maven.plugins</groupId>
               <artifactId>maven-deploy-plugin</artifactId>
               <configuration>
-                <updateReleaseInfo>true</updateReleaseInfo>
+                <altReleaseDeploymentRepository>${jboss.releases.repo.id}::${jboss.releases.repo.url}</altReleaseDeploymentRepository>
+                <altSnapshotDeploymentRepository>${jboss.snapshots.repo.id}::${jboss.snapshots.repo.url}</altSnapshotDeploymentRepository>
```

Comment:
This seems fine to me. It looks like you could just use `altDeploymentRepository` and it will use that if either of the other two are `null`.

Reply:
That's fair, but since we have separate releases and snapshots this seemed to be more correct. That said: there would be different behaviors since if someone gives `altDeploymentRepository` (rather than `altXxxxDeploymentRepository`) on the command line or in their POM, I think it would be ignored with this configuration. I'm OK with either solution (specifically, leaving this as-is versus changing `altReleaseDeploymentRepository` to `altDeploymentRepository`). WDYT?

Reply:
As Brian mentioned in chat, normally people should use the properties to customize the definition anyway... maybe.